proto/pom.xml

```diff
@@ -46,28 +46,45 @@
         </dependency>
     </dependencies>
 
-    <profiles>
-        <profile>
-            <id>build-protobuf</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.xolstice.maven.plugins</groupId>
-                        <artifactId>protobuf-maven-plugin</artifactId>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.github.os72</groupId>
+                <artifactId>protoc-jar-maven-plugin</artifactId>
+                <version>3.3.0</version>
```

Comment:
I went ahead and applied your suggestions.  Thanks, it's simplier than ever!  The only thing is the plugins are specific to the `proto` module so I didn't move them to `plugins-parent` otherwise they will be applied to all modules.

I'm looking forward to see this PR merged, so that #416 (support for snake case fields) can also be incorporated.

Also, please consider bumping the current version `6.1.0-SNAPSHOT` to eg `6.1.1-SNAPSHOT` to make sure developer working copies continue to build flawlessly.  I haven't included that change to this PR.